WalletWasabi.Fluent/Helpers/LabelHelpers.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Fluent.ViewModels.Wallets.Labels;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+public static class LabelHelpers
+{
+	public static IEnumerable<(string Label, int Score)> GetMostUsedLabels(this Wallet wallet, Intent intent)
```

Comment:
I'm not sure if this is the best name possible for the method. When I read it, it looks like it will return a plain list with the most used labels, but it's returned a ranking. Not a big deal, though.

Reply:
I agree with @SuperJMN on the naming.